### ISSUE TITLE
SF5 and SF6 supported only for private 0x12 syncword

### ISSUE THREAD

**Issue by reissjason** [Lora-net/sx1302_hal#124]
Would there be any issue changing SF5 and SF6 to use the public 0x34 syncword?

**Comment by mcoracin** [comment]
Yes, the 0x34 syncword cannot be coded for SF5, and for SF6 the sensitivity is degraded. That's why we kept 0x12 for public for SF5/SF6 in the hal.

**Comment by rsoss** [comment]
Hi @mcoracin does it mean that we must use the 0x12 (private syncword) for SF5/6? How would this interact with public syncword  (0x34) for SF7...12 on the same gateway?

**Comment by mcoracin** [comment]
Yes, you must use 0x12 for SF5/SF6.
The syncword configuration is per SF so it is not an issue to have SF5-6 with 0x12, and SF7-12 with 0x34.
The end-device has to use the proper syncword depending on the SF used for transmission.
